### ISSUE TITLE
Interesting literature with key points

### ISSUE THREAD

**Issue by MagnusOstertag** [MagnusOstertag/AI4ArcticSeaIceChallenge#4]
The research group of the challenge tells us:

- [AI4SeaIce: Toward Solving Ambiguous SAR extures in Convolutional Neural Networks for Automatic Sea Ice Concentration Charting](https://ieeexplore.ieee.org/document/9705586)
  - based on SAR it is a hard task for CNNs
    - the receptive field is important: 3068px
    - noise phenomenon is present in the Sentinel-1 ESA Instrument Processing Facility (IPF) v2.9 SAR data, particularly in subswath transitions, visible as long vertical lines and grained particles resembling small sea ice floes
  - U-Net CNN architecture plus symetrical blocks of convolutional, pooling and upsampling layers in the encoder and decoder of the U-Net
  - training on noise-corrected SAR data - NERSC
  - U-net code available at [github](https://github.com/astokholm/AI4SeaIce.git.)
  - There may be a trade-off between the level of detail and homogeneity—a larger receptive field increases the homogeneity of SIC predictions, but it also appears to reduce the level of detail in predictions
  - landfast ice predictions are still troublesome for the model
  - a suggestion in the literature is to increase the receptive field of the model, allowing it to predict a value of the output pixel, based on the information from a wider area in the input image.
  - preprocessing: Six scenes containing errors in the ice charts depicting open water as 100% sea ice are removed and scenes without sea ice are discarded to balance class distribution
  - testing: 23 scenes, deemed difficult by professional ice analysts, have been selected for testing.
  - class imbalance: how are the classes of sea ice concentration distributed? -> median frequency weighting scheme, weighted cross-entropy loss
  - In this study, SIC estimation is formulated as a classification problem with the (weighted) categorical cross-entropy as the loss function, which represents the dissimilarity between real distribution of labels and output distribution predicted by the model - instead of regression problem
  - there are random data augmentations of the dihedral group (rotation etc.) and affine transformations
  - training of about 80–90 epochs took approximately 22–24 h per model on two Nvidia TeslaV100 SXM2 32-GB graphics cards
  - instead of symmetrically doubling and halving the filters for each level, 16 in the initial and final levels, and 32 in the remaining levels are used. To both simplify and minimize the risk of overfitting to ambiguous SAR textures.
  - receptive field of the U-Net models is thus 44, 92, 188, 380, 764, 1532, and 3068 for 2–8 levels
  - The best performing model is number 10 with eight levels, a receptive field of 3068 pixels, and trained on NERSC noise correction, with a patch size of 768 × 768 with an overall R 2 -score of 86.34%
  - There may be a trade-off between the level of detail and homogeneity
- [AI4Artic challenge dataset manual]()
  - dataset description: the images are not equally distributed through the seasons, the incidence angle of the sensor affects the amout of radar backscatter
  - the texture is very important for the image classification
  - the sea maps are done via polygons, in which each individual grid cell may deviate substantially, but in average it is ok (-> smoothing of the result)
  - the regions near the ice edge are most important, and the ice maps there are more accurate and more detailed
  - the inter-analysts differences show a up to 20% deviation, especially in the intermediate concentrations
  - distance to land is especially useful to mitigate land spill-over effects at the coastal areas
  - NaN values are 2, non-data or masked pixels are 255

**Comment by MagnusOstertag** [comment]
Understanding U-nets:
- [Medical Image Segmentation Review: The success of U-Net](https://doi.org/10.48550/arXiv.2211.14830) (!)
- [U-Net and Its Variants for Medical Image Segmentation: A Review of Theory and Applications](https://doi.org/10.1109/ACCESS.2021.3086020)
   - attention u-net: can provide localized classification information as opposed to global classification
   - inception u-net (!): filter of multiple sizes on the same layer for images with large variations in shapes and sizes in the salient region. Can read high-level details across a spectrum of sizes and shapes without going too deep
   - recurrent u-net (!): can update the feature maps based on context from adjoining units
   - dense u-net: can have fewer channels as information is preserved btw layers
   - ensemble u-net: first high-level segmentation and then successive u-nets performing segmentation on smaller objects

Attention u-nets

- [Attention U-Net:
Learning Where to Look for the Pancreas](https://arxiv.org/abs/1804.03999)
   - automatically learns to focus on target structures of varying shapes and sizes
   - training behavior of the AGs can benefit from transfer learning and multi-stage training schemes

**Comment by MagnusOstertag** [comment]
Sea ice studies:

- [Architecture for Sentinel-1 and AMSR2 Data Fusion](https://ieeexplore.ieee.org/document/9133205)
  - image segmentation like CNN
  - fusion of data from SAR and AMSR2
- [A labelled ocean SAR imagery dataset of ten geophysical phenomena from Sentinel-1 wave mode](https://rmets.onlinelibrary.wiley.com/doi/full/10.1002/gdj3.73) and others
- [Construction of a climate data record of sea surface temperature from passive microwave measurements](https://www.sciencedirect.com/science/article/abs/pii/S0034425719305048)
  - measure the sea temperature

**Comment by MagnusOstertag** [comment]
More than u-nets:

- [CR-U-Net: Cascaded U-Net with Residual Mapping for Liver Segmentation in CT Images](https://doi.org/10.1109/VCIP47243.2019.8966072)
- [Automatic Brain Structures Segmentation Using Deep Residual Dilated U-Net](https://doi.org/10.1007/978-3-030-11723-8_39): captures rich context information of different tissues for the segmentation of eight brain structures

**Comment by MagnusOstertag** [comment]
Recommended papers in the webinar (if also unknown):

* [Sea Ice Concentration Estimation: Using Passive Microwave and SAR Data With a U-Net and Curriculum Learning](https://doi.org/10.1109/JSTARS.2021.3076109)
* [Prediction of Categorized Sea Ice Concentration From Sentinel-1 SAR Images Based on a Fully Convolutional Network](https://doi.org/10.1109/JSTARS.2021.3074068)
* [Classification of Sea Ice Types in Sentinel-1 SAR Data Using Convolutional Neural Networks](https://doi.org/10.3390/rs12132165)
* [A Convolutional Neural Network Architecture for Sentinel-1 and AMSR2 Data Fusion](https://doi.org/10.1109/TGRS.2020.3004539)
